extensions/wikia/Chat2/ChatBanListSpecial_helper.php

```diff
@@ -197,21 +197,20 @@ public function loadData() {
 	private function getUserLinks( $user ) {
 
 		$userIsBlocked = $this->wg->User->isBlocked( true, false );
-		$oEncUserName = urlencode( $user->getName() );
+		$username = $user->getName();
 		$links = [
 			0 => "",
 			1 => Linker::link(
-				Title::newFromText( 'Contributions', NS_SPECIAL ),
-				$this->wg->Lang->ucfirst( wfMsg( 'contribslink' ) ),
-				[ 'target' => $oEncUserName]
+				SpecialPage::getSafeTitleFor( 'Contributions', $username ),
+				$this->wg->Lang->ucfirst( wfMsg( 'contribslink' ) )
```

Comment:
I think we can get rid of the `ucfirst` here because there is no other MediaWiki page (compare with e.g. BlockList) that does this kind of stupid capitalization.